src/libsystemd-network/sd-radv.c

```diff
@@ -197,7 +203,19 @@ static int radv_send(sd_radv *ra, const struct in6_addr *dst,
                 msg.msg_iovlen++;
         }
 
-        LIST_FOREACH(prefix, p, ra->prefixes) {
+        LIST_FOREACH_SAFE(prefix, p, next, ra->prefixes) {
```

Comment:
LIST_FOREACH_SAFE is for the case where the list can be modified during traversal. This doesn't seem to be the case here.

Reply:
Yes, LIST_FOREACH it is.